_posts/2022-01-18-dotnet-core-query.adoc

```diff
@@ -0,0 +1,31 @@
+---
+layout: blog
+title: Querying Infinispan with the new .NET core client
+permalink: /blog/:year/:month/:day/dotnet-core-query
+date: '2022-01-18T12:00:00.000-00:00'
+author: rigazilla
+tags: [ "hotrod", ".Net Core", "client", "C#", "query", "query" ]
+---
+
+Dear Infinispanners and .NET Core lovers,
+
+time is passed, keys have been pressed and features have been implemented since
+we https://infinispan.org/blog/2021/06/21/infinispan-dotnet-core-client[presented],
+some months ago, the new .NET core client project.
+
+To show the progresses done on the project we decided to write a demo/tutorial on
+how you can run remote queries against Infinispan in your .NET Core application.
+
+The demo aims to proof the overall maturity of the client and highlight some key client features like:
+
+* it's shipped as a nuget package, so it can be easily included in a C# project;
+* it's based on non blocking core, thus it has a native support for concurrency;
```

Comment:
```suggestion
* Using Infinispan's non-blocking core to add native support for concurrency.
```

Reply:
@oraNod, I meant the client uses BeetleX non-blocking library as its core. Maybe we could say:
` * Using [BeetleX's](https://github.com/beetlex-io/BeetleX) non-blocking core to add native support for concurrency.

Reply:
@rigazilla Thanks I updated the suggestion with that.